frontend/src/routes/+layout.svelte

```diff
@@ -37,7 +37,9 @@
         <span class="block text-sm text-gray-900 dark:text-white"> Kaori Chihiro </span>
         <span class="block truncate text-sm font-medium"> kaori@peerrequest.com </span>
       </div>
-      <DropdownItem>My Papers</DropdownItem>
+      <DropdownItem>
+        <a href="/my-papers">My Papers</a>
+      </DropdownItem>
```

Comment:
You solution is good but clicking like shown below will not work because the anchor (`<a>...</a>`) is only wrapping the text.

![Screenshot from 2023-01-24 23-12-49](https://user-images.githubusercontent.com/25325997/214432946-f5c612c2-ed02-4083-b7ff-272bf23a100b.png)

Consider wrapping the anchor around the whole dopdown entry.

```suggestion
      <a href="/my-papers">
        <DropdownItem>
          My Papers
        </DropdownItem>
      </a>
```